a3p-integration/proposals/z:acceptance/package.json

```diff
@@ -9,6 +9,7 @@
   "type": "module",
   "license": "Apache-2.0",
   "dependencies": {
+    "@agoric/ertp": "0.16.3-u17.1",
```

Comment:
this doesn't match the `@agoric/internal` version so there are duplicates.

Please make them both`"dev"` so they're testing the latest master push. The version won't change unless the `yarn.lock` changes, which can't happen without a PR, and that would have to pass CI to merge.

Reply:
made both of them `dev` and re-enforced `yarn.lock` by doing;
```
cd proposals/z:acceptance
rm yarn.lock
touch yarn.lock 
yarn install
```

addressed in https://github.com/Agoric/agoric-sdk/commit/712f97060737ce2a54d4e27ba2047142da6585ab